yw_website/.gitignore

```diff
@@ -15,6 +15,7 @@
 
 # Generated files
 .idea/**/contentModel.xml
+.idea/
```

Comment:
we might just want to ignore all .idea files: `.idea/**`

Reply:
We have to ignore the folder as well, not just the contents